pyccl/tests/test_baryons_mead20.py

```diff
@@ -0,0 +1,136 @@
+"""Tests for HMCode/Mead2020 nonlinear P(k) with baryonic feedback."""
+
+import pytest
+
+import numpy as np
+
+import pyccl as ccl
+
+
+camb = pytest.importorskip("camb")
+
+TOL_MEAD20 = 3e-5
```

Comment:
Why is this so high? Shouldn't we be getting much better agreement given that we should be calling camb in exactly the same way?

Reply:
This tolerance is inherited from the existing test_nonlin_camb_power test, which already used rtol=3e-5 for CAMB+Mead2020 vs CCL

In local runs the max relative difference between CAMB’s nonlinear P(k) and CCL’s matter_power_spectrum="camb" with halofit_version="mead2020_feedback" is at the ~1e-5 level once we restrict to a safe k-range, so 3 times that  is just a small safety margin for interpoltaion / platform differences

I can try tightening it to 1e-5 and see if CI is stable; if that passes I’m happy to lower the tolerance